src/components/molecules/widgets/HeadOnCollisionsComparisonWidget.tsx

```diff
@@ -38,27 +38,28 @@ const useStyles = makeStyles(() => ({
     display: 'flex',
     height: MAIN_CONTENT_HEIGHT,
     width: '100%',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    fontSize: `${(PROPORTION * PRIMARY_FONT_SIZE).toString()}px`,
```

Comment:
1. this can be constant.
2. why not using typography components? they should not be mixed with css

Reply:
Move label fontsize into pie-chart-view